PSKoans/Koans/Introduction/AboutNumbers.Koans.ps1

```diff
@@ -61,6 +61,15 @@ Describe 'Basic Number Types' {
 Describe "Banker's Rounding" {
 
     It 'rounds to nearest even number on .5' {
+        <#
+            Rounding
+
+            The rounding used in powershell is called "Rounding to Even" or "Banker's Rounding".
+            Numbers will be rounded to the nearest _even_ Integer.
+
+            This Method stems from the underlying .NET libraries and can be found in more detail here:
```

Comment:
```suggestion
            This behaviour stems from the underlying .NET library method [math]::Round() which is documented in more detail here:
```

Reply:
Changed in 15d69579941977f82371cfe890290cf8fcd0168c